keda/manifest.yaml

```diff
@@ -1,7 +1,6 @@
----
 name: keda
 title: "KEDA"
-version: "2.9.2"
+version: "2.9.3"
```

Comment:
@olblak is should be 2.9.4 right? but the manifest is update to 2.9.3 instead.

Reply:
By looking at the helm chart repository 

https://github.com/kedacore/charts/blob/main/keda/Chart.yaml\

It appears that the chart version is 2.9.4 and the application 2.9.3

Reply:
My understanding is the manifest shows the application version installed by the helm chart

Reply:
I am now wondering if It wouldn't be better to monitor this file https://github.com/kedacore/charts/blob/main/keda/Chart.yaml directly similarly to the manifest that I wrote for argocd